packages/integration-karma/scripts/shared/options.js

```diff
@@ -14,16 +14,16 @@ if (process.env.NATIVE_SHADOW) {
 
 const COMPAT = Boolean(process.env.COMPAT);
 const DISABLE_SYNTHETIC = Boolean(process.env.DISABLE_SYNTHETIC);
-const FORCED_NATIVE_SHADOW_MODE_FOR_TEST = Boolean(process.env.FORCED_NATIVE_SHADOW_MODE_FOR_TEST);
+const FORCE_NATIVE_SHADOW_MODE_FOR_TEST = Boolean(process.env.FORCE_NATIVE_SHADOW_MODE_FOR_TEST);
 const TAGS = [`${DISABLE_SYNTHETIC ? 'native' : 'synthetic'}-shadow`, COMPAT && 'compat'].filter(
-    (v) => Boolean(v)
+    Boolean
 );
 
 module.exports = {
     // Test configuration
     COMPAT,
-    DISABLE_SYNTHETIC,
-    FORCED_NATIVE_SHADOW_MODE_FOR_TEST,
+    FORCE_NATIVE_SHADOW_MODE_FOR_TEST,
+    SYNTHETIC_SHADOW_ENABLED: !DISABLE_SYNTHETIC,
```

Comment:
minor: can we use DISABLE_SYNTHETIC directly here? Then NATIVE_SHADOW in env.js would be `NATIVE_SHADOW: ${DISABLE_SYNTHETIC || FORCE_NATIVE_SHADOW_MODE_FOR_TEST}`.

Reply:
`options.js` is basically the source of truth for logic that requires input from environment variables so I think we should handle normalization once here.